contrib/cmd/recvtty/recvtty.go

```diff
@@ -105,7 +105,10 @@ func handleSingle(path string, noStdin bool) error {
 	if err != nil {
 		return err
 	}
-	console.ClearONLCR(c.Fd())
+	err = console.ClearONLCR(c.Fd())
+	if err != nil {
```

Comment:
```suggestion
	if err := console.ClearONLCR(c.Fd()); if err != nil {
```

Reply:
Suggestion applied in the squashed commit.